### ISSUE TITLE
Suggest: Do not switch scene tabs in editor on right click

### ISSUE THREAD

**Issue by Xtarsia** [godotengine/godot-proposals#9353]
### Describe the project you are working on

open world game with some large data heavy scenes.

### Describe the problem or limitation you are having in your project

after swapping to a different scene, if I want to close out another tab, right clicking swaps focus and loads the scene. this can cause the editor to freeze whilst it loads, which is normally fine if wanting to swap to the scene, however if all i want to do is close the tab, this is annoying.

the menu doesnt even show up until the scene is loaded.
![image](https://github.com/godotengine/godot-proposals/assets/69606701/15b4a272-5d3d-405d-b7b1-c9807ac65624)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

allowing the right click context menu to load switchingthe active tab would be make things smoother.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

only the load the context menu, when right clicking. left click only should swap tab focus.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

core editor functionality

### Is there a reason why this should be core and not an add-on in the asset library?

core editor functionality

**Comment by kitbdev** [comment]
Yeah, it works based on the currently opened scene and the other menu options would need some changing to work on non active scenes.

As a workaround, you can use the middle mouse button to close the scene without switching to it, or the 'Close Tabs to the Right' option on the current tab if your tabs are ordered like that.
